### ISSUE TITLE
Timeline: Items out of the range are displayed when using "min" option  

### ISSUE THREAD

**Issue by ocurero** [almende/vis#1609]
Hi,

I just discovered a bug that happens if I use the "min" option to limit the date frame of the timeline. This is how it looks.

![timeline](https://cloud.githubusercontent.com/assets/2606647/12619540/c1d19df0-c518-11e5-84ee-b36974222ee1.png)

This code (taken from the limit_move_and_zoom example) works fine:

```javascript
  // create some items
  // note that months are zero-based in the JavaScript Date object, so month 4 is May
  var items = new vis.DataSet([
    {'start': new Date(2012, 4, 25), 'content': 'First'},
    {'start': new Date(2012, 4, 26), 'content': 'Last'}
  ]);

  // create visualization
  var container = document.getElementById('visualization');
  var options = {
    height: '300px',
    min: new Date(2012, 0, 1),                // lower limit of visible range
    max: new Date(2013, 0, 1),                // upper limit of visible range
    zoomMin: 1000 * 60 * 60 * 24,             // one day in milliseconds
    zoomMax: 1000 * 60 * 60 * 24 * 31 * 3     // about three months in milliseconds
  };

  // create the timeline
  var timeline = new vis.Timeline(container);
  timeline.setOptions(options);
  timeline.setItems(items);
```

However, this one will make the item "First" to be displayed at the left side of the timeline because we have changed its date to be 2000:
```javascript
  // create some items
  // note that months are zero-based in the JavaScript Date object, so month 4 is May
  var items = new vis.DataSet([
    {'start': new Date(2000, 4, 25), 'content': 'First'},
    {'start': new Date(2012, 4, 26), 'content': 'Last'}
  ]);

  // create visualization
  var container = document.getElementById('visualization');
  var options = {
    height: '300px',
    min: new Date(2012, 0, 1),                // lower limit of visible range
    max: new Date(2013, 0, 1),                // upper limit of visible range
    zoomMin: 1000 * 60 * 60 * 24,             // one day in milliseconds
    zoomMax: 1000 * 60 * 60 * 24 * 31 * 3     // about three months in milliseconds
  };

  // create the timeline
  var timeline = new vis.Timeline(container);
  timeline.setOptions(options);
  timeline.setItems(items); 
```

**Comment by josdejong** [comment]
Thanks for the clear problem description. We've just released v4.12.0 which resolves this issue (thanks @ludost). 

However v4.12.0 also introduced a regression which will let your code fail: it throws an error right when you don't provide a third argument options to the Timeline constructor. We will publish a bug fix release for this within a few days.

**Comment by ocurero** [comment]
Jos,

My original issue is fixed with v4.13.0 but the error that you talk about is still not fixed (though it's very easy to solve).

Thanks,

**Comment by josdejong** [comment]
Thanks for the feedback. I've just released v4.14.0 which has the constructor issue fixed.
